src/errors.ts

```diff
@@ -33,8 +33,8 @@ export class AlertError extends Error {
 
     public matches(errorMsg: string, config: ClientConfig): boolean {
         const pattern = this.opts.pattern
-            .replace('$rendezvousURL', config.rendezvousURL)
-            .replace('$transitRelayURL', config.transitRelayURL);
+            .replace('$rendezvousURL', config.rendezvousURL!)
+            .replace('$transitRelayURL', config.transitRelayURL!);
```

Comment:
I am concerned by the need to put an exclamation mark, which means, we are pretty sure at this point in code that those two variables (`rendezvousURL` and `transitRelayURL`) are not undefined. This would not be true if we are running the application on `localhost` during development.